workflows/cve-fixer/.claude/commands/cve.find.md

```
@@ -51,59 +51,40 @@ Report: artifacts/cve-fixer/find/cve-issues-20260226-145018.md
+   TEST_RESPONSE=$(curl -s -o /dev/null -w "%{http_code}" -X GET \
+     --connect-timeout 10 --max-time 15 \
+     -H "Authorization: Basic ${AUTH}" \
+     -H "Content-Type: application/json" \
+     "${JIRA_BASE_URL}/rest/api/3/myself")
+   ```
+
+   - **HTTP 200** → credentials valid, proceed
+   - **HTTP 401** → credentials missing or invalid. Only now inform the user:
+     - Check if `JIRA_API_TOKEN` and `JIRA_EMAIL` are configured as Ambient session secrets
+     - If not, generate a token at https://id.atlassian.com/manage-profile/security/api-tokens and export:
+       ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)